core/services/head_broadcaster.go

```diff
@@ -83,15 +83,18 @@ func (hr *HeadBroadcaster) Connect(head *models.Head) error {
 func (hr *HeadBroadcaster) Disconnect() {}
 
 func (hr *HeadBroadcaster) OnNewLongestChain(ctx context.Context, head models.Head) {
-	hr.mailbox.Deliver(head)
+	wasOverCapacity := hr.mailbox.Deliver(head)
+	if wasOverCapacity {
+		logger.Error("HeadBroadcaster: head mailbox is over capacity - dropped the oldest unprocessed head")
```

Comment:
This probably shouldn't be an `Error` level log. I would make it `Debug`, as this is common and expected and not a reason for alarm.

Reply:
oh right, it has capacity of 1. So i'll remove this logging